incubator/appdaemon/0.1.0/test_values.yaml

```
@@ -0,0 +1,42 @@
+image:
+  repository: acockburn/appdaemon
+  pullPolicy: IfNotPresent
+  tag: "4.0.8"
+
+strategy:
+  type: Recreate
+
+# Configure the Security Context for the Pod
+podSecurityContext:
+  runAsNonRoot: false
+  runAsUser: 0
+  runAsGroup: 0
+  fsGroup: 0
+
+
+env:
+  URL: ""
+  Token: "" This is a Statping instance deployed as Helm chart
+
+
+services:
+  main:
+    port:
+      port: 5050
+  tcp:
+    enabled: true
+    type: ClusterIP
+    port:
+      port: 51050
+      protocol: TCP
+      targetPort: 51050
+
+ingress:
+  enabled: false
+
+persistence:
+  data:
+    enabled: false
+    emptyDir:
```

tests often want this to be attatched, try using:
```suggestion
    enabled: true
    emptyDir: true
```